README.md

```diff
@@ -21,9 +21,6 @@ from source
     cd personal-backend
     pip install .
 
-from pip
-
-    pip install personal-mycroft-backend
```

Comment:
Why not changing it to point to this git instead of deleting it;

pip install git+https://github.com/MycroftAI/personal-backend.git

Reply:
makes sense